src/Fixer/ArrayNotation/AlphabeticalArrayKeySortFixer.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ArrayNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\CT;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Abdurrahman Uymaz <abdurrahman.uymaz@mobian.global>
+ * @author Lars Grevelink <lars.grevelink@mobian.global>
+ * @author Leander Philippo <leander.philippo@mobian.global>
+ */
+final class AlphabeticalArrayKeySortFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        // Return a definition of the fixer, it will be used in the documentation.
+        return new FixerDefinition(
+            'Sorts keyed arrays by alphabetical order.',
```

Comment:
```suggestion
            'Sorts keyed arrays alphabetically.',
```
Isn't that better English?

Reply:
I would like to point that instead of `keyed` we should use `associative`, because it's more appropriate 🙂.

```suggestion
            'Sorts associative arrays by keys using alphabetic order.',
```

Reply:
It sorts arrays like `[1=>0, 3=>0, 2=>0]` and AFAIK they don't count as associative. Or do they?

Reply:
So let's remove `keyed` part, because every array has keys 😉.

Reply:
Every array does, but not every array's definition - `[3, 1, 2]` won't be touched.

Reply:
You're right 👍, did not think this way.